### PR TITLE
add SLF4J and test logs

### PR DESCRIPTION
**Pull request by wiolaSzczepanik**

### CODE REVIEW

File: backend/src/main/java/com/swingevents/SwingEvents/SwingEventsApplication.java [hunk 1]
```diff
@@ -9,8 +12,11 @@
 @SpringBootApplication
 public class SwingEventsApplication {
 
+    public static Logger LOGGER = LoggerFactory.getLogger(Slf4j.class);
```
Comment (Mequrel) [inline backend/src/main/java/com/swingevents/SwingEvents/SwingEventsApplication.java]: public static final

Comment (Mequrel) [reply backend/src/main/java/com/swingevents/SwingEvents/SwingEventsApplication.java]: Also the class should be the class the logger is declared in, so SwingEventsApplication in this case

File: backend/src/main/resources/log4j.properties [hunk 2]
```diff
@@ -0,0 +1,5 @@
+log4j.rootLogger=DEBUG, STDOUT
```
Comment (Mequrel) [inline backend/src/main/resources/log4j.properties]: please set up root logger to INFO

File: backend/pom.xml [hunk 3]
```diff
@@ -81,6 +81,14 @@
 			<artifactId>commons-io</artifactId>
 			<version>2.6</version>
 		</dependency>
+		<!-- https://mvnrepository.com/artifact/org.slf4j/slf4j-log4j12 -->
+		<dependency>
+			<groupId>org.slf4j</groupId>
+			<artifactId>slf4j-log4j12</artifactId>
```
Comment (Mequrel) [inline backend/pom.xml]: Why log4j 1.2? Please use log4j 2

File: backend/src/main/java/com/swingevents/SwingEvents/SwingEventsApplication.java [hunk 4]
```diff
@@ -1,5 +1,8 @@
 package com.swingevents.SwingEvents;
 
+import lombok.extern.slf4j.Slf4j;
```
Comment (Mequrel) [inline backend/src/main/java/com/swingevents/SwingEvents/SwingEventsApplication.java:3]: If you already use lombok you can leverage this functionality:
https://projectlombok.org/api/lombok/extern/slf4j/Slf4j.html

**Review by Mequrel — CHANGES_REQUESTED** [review]

File: backend/src/main/java/com/swingevents/SwingEvents/SwingEventsApplication.java (hunk 1 above)
Comment (wiolaSzczepanik) [reply backend/src/main/java/com/swingevents/SwingEvents/SwingEventsApplication.java]: Done.

File: backend/src/main/resources/log4j.properties (hunk 2 above)
Comment (wiolaSzczepanik) [reply backend/src/main/resources/log4j.properties]: Done.

File: backend/pom.xml (hunk 3 above)
Comment (wiolaSzczepanik) [reply backend/pom.xml]: Done.

**Pull request merged**